src/base/tests/test_util.py

```diff
@@ -1436,6 +1437,8 @@ def not_doing_anything_converter(el):
 
 class TestHTMLFormat(UnitTestCase):
     def testsnip(self):
+        cr = self.registry.cursor()
```

Comment:
>  Is it because now you commit the cursor?

Yes!

> IIRC we have other examples in testing utils were we instead remove the commit restriction from test cursors.

True, but this approach only works with threads, not with forked processes, I think. Also, the approach I chose is the more common approach looking at tests in the `odoo` repo. It also feels cleaner to me, personally.